gcp/private-agent.tf

```diff
@@ -0,0 +1,150 @@
+# Target Pool for external load balancing access
+resource "google_compute_target_pool" "agent-pool" {
+  name = "${data.template_file.cluster-name.rendered}-agent-pool"
+
+  instances = ["${google_compute_instance.agent.*.self_link}"]
+}
+
+resource "google_compute_instance_group" "agent" {
+  name        = "${data.template_file.cluster-name.rendered}-agent-cluster"
+  description = "DC/OS Agent Instance Group"
+
+  instances = [
+    "${google_compute_instance.agent.*.self_link}"
+  ]
+
+  named_port {
+    name = "http"
+    port = "80"
+  }
+
+  named_port {
+    name = "https"
+    port = "443"
+  }
+
+  named_port {
+    name = "mesos-agent"
+    port = "5051"
+  }
+
+  named_port {
+    name = "ssh"
+    port = "22"
+  }
+
+  zone = "${data.google_compute_zones.available.names[0]}"
+}
+
+# deploy image
+resource "google_compute_instance" "agent" {
+   name         = "${data.template_file.cluster-name.rendered}-agent-${count.index + 1}"
+   machine_type = "${var.gcp_agent_instance_type}"
+   zone         = "${data.google_compute_zones.available.names[0]}"
+   count        = "${var.num_of_private_agents}"
+ 
+  labels {
+   owner = "${coalesce(var.owner, data.external.whoami.result["owner"])}"
+   expiration = "${var.expiration}"
+   name = "${data.template_file.cluster-name.rendered}-agent-${count.index + 1}"
+   cluster = "${data.template_file.cluster-name.rendered}"
+  }
+
+  boot_disk {
+    initialize_params {
+      image = "${module.dcos-tested-gcp-oses.gcp_image_family}/${module.dcos-tested-gcp-oses.gcp_image_name}"
+      size  = "${var.instance_disk_size}"
+    }
+  }
+
+  connection {
+    # host = "${element(google_compute_instance.agent.*.network_interface.0.access_config.0.assigned_nat_ip, count.index)}"
+    user = "${coalesce(var.gce_ssh_user, module.dcos-tested-gcp-oses.user)}"
+  }
+
+  network_interface {
+    subnetwork = "${google_compute_subnetwork.private.name}"
+    access_config {
+        // IP
```

Comment:
same here, remove the comment or add meaningful description

Reply:
done! tfvm